### DISCUSSION TITLE
[Config Support]: [ERROR] Frigate config file not found

### DISCUSSION THREAD

**Discussion by voltlog** [blakeblackshear/frigate#10741] [Config Support] [answered]
### Describe the problem you are having

Trying to run Frigate on Debian 12. Created the directory structure as shown [in the documentation](https://docs.frigate.video/guides/getting_started) and the necessary config files.
```
.
├── docker-compose.yml
├── config/
│   └── config.yml
└── storage/
```
These reside on this path _/home/frigate/frigate-docker_

When I try to start the container it is continuously restarting and I find this in the logs: _[ERROR] Frigate config file not found_

Each time I start the container, I always find an empty directory called "config.yml" created under /home/frigate/frigate-docker

What am I doing wrong here?

Thanks!

### Version

NA

### Frigate config file

```yaml
volumes:
      - /etc/localtime:/etc/localtime:ro
      - ./config:/config
      - ./storage:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
```

### Relevant log output

```shell
2024-03-29 19:08:16.261412383  [INFO] Starting NGINX...
2024-03-29 19:08:16.263868050  [INFO] Preparing Frigate...
2024-03-29 19:08:16.264741038  [ERROR] Frigate config file not found
2024-03-29 19:08:16.304658449  [INFO] Service Frigate exited with code 1 (by signal 0)
s6-rc: info: service legacy-services successfully started
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service nginx: stopping
s6-rc: info: service go2rtc-healthcheck: stopping
2024-03-29 19:08:16.366574146  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
s6-rc: info: service go2rtc-healthcheck successfully stopped
2024-03-29 19:08:16.408586127  [INFO] Service NGINX exited with code 0 (by signal 0)
s6-rc: info: service nginx successfully stopped
s6-rc: info: service nginx-log: stopping
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

CPU (no coral)

**Comment by NickM-27** [comment, marked as answer]
using absolute paths is recommended instead of relative paths which can depend on where you are running docker compose from

**Comment by voltlog** [comment]
This was fixed by putting the full system path in the config and running docker-compose again.
Thank you!
